src/style.css

```diff
@@ -0,0 +1,136 @@
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.card-container {
+  display: flex;
+}
+
+.card-product {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+  height: 406px;
+}
+
+.card-product__image {
+  margin-top: 32px;
+  margin-left: 19px;
+  height: 134px;
+  width: 160px;
+}
+
+.card-product__image_size {
+  height: 134px;
+  width: 160px;
+}
+
+.card-product__name {
+  margin-top: 40px;
+  margin-left: 16px;
```

Comment:
don't repeat yourself
instead of setting margins for each elements inside card, just set paddings for the whole card